docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,174 @@
+attempt to change speeds faster than is safe.
+
+There are two parameters in the :ref:`Brushless Drive client <brushless_drive>` that affect your module's slew rate limiting:
+
+* ``slew_volts_per_second`` defines the actual slew rate in volts per second applied to voltage commands. This defines the strength of the filter's impact.
+* ``slew_enable`` determines whether or not ``slew_volts_per_second`` is actually applied to voltage commands. If ``slew_enable`` is false, then there will not be slew limiting applied.
+
+The following examples are meant to illustrate the effect that slew limiting has on module control. All examples start with the module in coast followed by a voltage step command to 2V.
+
+1. Slew limiting disabled
+
+.. image:: ../_static/manual_images/safety/slew_rate_off.png
+    :scale: 80%
+
+2. Slew limit enabled at 10V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_10vps.png
+    :scale: 80%
+
+3. Slew limit enabled at 5V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_5vps.png
+    :scale: 80%
+
+4. Slew limit enabled at 2.5V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_2_5vps.png
+    :scale: 80%
+
+5. Slew limit enabled at 1V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_1.png
+    :scale: 80%
+
+As you can see, the slew limit parameter helps protect against sudden changes in commanded voltage. This can be especially useful in :ref:`noisy/vibration heavy 
+environments <vibration_and_jittering>`. The amount of slew filtering that is necessary (if any) is completely application specific, and will require manual tuning.
+
+Brushless Drive Current Limiter
+====================================
+:ref:`Brushless Drive's <brushless_drive>` ``motor_I_max`` defines that maximum allowable drive current. In general, this current limit protects your module against extreme step commands, and ensures that the current moving through the motor is always in a safe region.
+This behavior is illustrated in the examples below. The first is the module with a ``motor_I_max`` of 50A. It is given a step to 5V, then after 5 seconds is stepped back to a 0V control.
+
+.. image:: ../_static/manual_images/safety/current_limit_50.png
+    :scale: 80%
+
+Now, we set the current limit to 0.5A. You will see that the motor's rate of change is much slower. As mentioned above, this is meant to decrease the likelihood of dangerous currents as the module steps between velocities.
+
+.. image:: ../_static/manual_images/safety/current_limit_0_5.png
+    :scale: 80%
+
+The biggest difference between this current limit and :ref:`slew_rate` is that the slew rate parameter defines an absoulte linear relation between the control voltage applied and 
+the time it takes to get there. This current limit is not meant to limit the rate of change of commanded voltages, it is meant to protect against sudden voltage 
+spikes due to large voltage steps.
```

Comment:
There's a bit more to it than this, as I touched on in my comment above. The current max limit does a little more for us than just controlling steps like the slew limit, it more generally protects us from high current events, like a stall. For a colorful example, I once threw a towel at a spinning motor to test its I Max. We didn't change its command or drive voltage at all, it just couldn't spin, and so no more back-EMF and so loads of current. Maybe that's what you were going for here as well when you say "voltage steps", but it may be good to clarify that difference of it a bit more. I'm happy to sit down and talk about this more in person if you want some more insight on what the differences are